python/pyspark/ml/classification.py

```diff
@@ -51,6 +51,8 @@
            'BinaryRandomForestClassificationTrainingSummary',
            'NaiveBayes', 'NaiveBayesModel',
            'MultilayerPerceptronClassifier', 'MultilayerPerceptronClassificationModel',
+           'MultilayerPerceptronClassificationSummary',
+           'MultilayerPerceptronClassificationTrainingSummary',
```

Comment:
@huaxingao, are these going to be documented? Might be good to list at `pyspark.ml.rst` too if so. The documentation PR (https://github.com/apache/spark/pull/29188) was merged.

Reply:
@HyukjinKwon updated `pyspark.ml.rst`. Thanks!